packages/mobile/src/home/WalletHome.tsx

```diff
@@ -72,12 +78,15 @@ const mapStateToProps = (state: RootState): StateProps => ({
   recipientCache: phoneRecipientCacheSelector(state),
   appConnected: isAppConnected(state),
   numberVerified: state.app.numberVerified,
+  cashInButtonExpEnabled: state.app.cashInButtonExpEnabled,
```

Comment:
Can you please make a selector for this in the `selectors.ts` file within the `app` directory? While the approach you've chosen is fine, we generally want to define selectors and import that in rather than define them inline everywhere. This makes things easier as we continue to refactor class components into functional components

Reply:
idk if I see a real benefit of that. My feeling is those remote config flag will eventually go away. If the experiment go well, then we will ramp it to 100%, which means the logic will live in the app forever. If the experiment doesn't go well, then we will delete the code logic, including the flag here.

Reply:
I see your point in this specific case. If you don't want to make this change given the code will be deleted either way, that's ok but please be aware that this is not how we want to access Redux state elsewhere in the app